chai-http/index.d.ts

```diff
@@ -81,5 +81,6 @@ declare namespace ChaiHttp {
 
 declare module "chai-http" {
 	function chaiHttp(chai: any, utils: any): void;
+	namespace chaiHttp {}
```

Comment:
Indeed, this PR is bad, yet I can not use chat-http with ES6 modules, I'm forced to use commonjs since `import * as chaiHttp from "chai-http";` says that chai-http is not a valid module